modal/app.py

```diff
@@ -607,6 +607,7 @@ def function(
         region: Optional[Union[str, Sequence[str]]] = None,  # Region or regions to run the function on.
         enable_memory_snapshot: bool = False,  # Enable memory checkpointing for faster cold starts.
         block_network: bool = False,  # Whether to block network access
+        restrict_modal_access: bool = False,  # Whether to allow this function to access other Modal resources
```

Comment:
```suggestion
        restrict_modal_access: bool = False,  # Whether to allow this function access to other Modal resources
```

tripped over the double infinitive the first time i read it (if you want to make this change, good to change in the cls signature too)